src/sdk/resources/policyItemTrackingGroups.ts

```diff
@@ -0,0 +1,9 @@
+import { IHttpClient } from '../http';
```

Comment:
Tracking groups would be retrieved from the policy `groups` property. (i.e. `/policies/{id}?fields=groups`)